src/lib/libast/features/syslocale.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
This fails on FreeBSD 11 because bash is installed as */usr/local/bin/bash*. It doesn't look like this depends on bash behavior so you could change this to `#!/bin/sh`. Alternatively, `#!/usr/bin/env bash`.